students/duanez2021/lesson02/fizzBizz.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+import sys
+##############################################################
+#
+# 20200603    djm Duane McCollum,
+#
+#  https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/fizz_buzz.html
+#
+#  Write a program that prints the numbers from 1 to 100 inclusive.
+#  But for multiples of three print “Fizz” instead of the number.
+#  For the multiples of five print “Buzz” instead of the number.
+#  For numbers which are multiples of both three and five print “FizzBuzz” instead.
+
+##############################################################
+
+def fizzBuzz():
```

Comment:
In Python function names (and almost everything else besides classes) should be snake_case.   So, fizz_buzz():